packages/sdk/README.md

```diff
@@ -156,6 +156,187 @@ channel.banUser(message.sender);
 
 To verify if the user was banned, check the property `isBanned` in the user object.
 
+### Q&A
+
+If the "Q&A" feature is enabled for your chat plan and activate for the current chat, you can implement it using this module. 
```

Comment:
We don't need to say this here because users of the SDK are paying customers that will always have the Q&A enaabled.
Suggestion:
"When a Q&A session is enabled for the Chat Room, you can implement the basic Q&A functionality using this module."